app/screens/Onboarding/Parents/LanguageSelection/LanguageSelection.data.ts

```diff
@@ -1,6 +1,6 @@
 import { IRadioButtonListItem } from '../../../../components';
 
-export const LANGUAGES: ReadonlyArray<IRadioButtonListItem> = [
+export const LANGUAGES: IRadioButtonListItem[] = [
```

Comment:
ReadonlyArray is needed, so leave only `id` and `title`
The `isActive` you need map later with these Readonly values